### PR TITLE
boards: arm: atmel: Update documentation

### PR DESCRIPTION
**Pull request by nandojve**
- Update feature table
- Update IC variations
- Update broken links

### CODE REVIEW

**Review by galak — APPROVED** [review]

File: boards/arm/sam_v71_xult/doc/index.rst [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+SAM V71(B) Xplained Ultra
+#########################
 
 Overview
 ********
 
 The SAM V71 Xplained Ultra evaluation kit is a development platform to
-evaluate the Atmel SAM V71 series microcontrollers.
+evaluate the Atmel SAM V71 series microcontrollers. The current version
+allow to use both IC variations ATSAMV71Q21A(B).
```
Comment (ioannisg) [inline boards/arm/sam_v71_xult/doc/index.rst:11]: allows

File: boards/arm/sam_e70_xplained/doc/index.rst [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-################
+SAM E70(B) Xplained
+###################
 
 Overview
 ********
 
 The SAM E70 Xplained evaluation kit is a development platform to evaluate the
-Atmel SAM E70 series microcontrollers.
+Atmel SAM E70 series microcontrollers. The current version allow to use both
```
Comment (ioannisg) [inline boards/arm/sam_e70_xplained/doc/index.rst:10]: allows

**Review by ioannisg — CHANGES_REQUESTED** [review]

File: boards/arm/sam_v71_xult/doc/index.rst (hunk 1 above)
Comment (nandojve) [reply boards/arm/sam_v71_xult/doc/index.rst]: Tks, done!

File: boards/arm/sam_e70_xplained/doc/index.rst (hunk 2 above)
Comment (nandojve) [reply boards/arm/sam_e70_xplained/doc/index.rst]: again, tks

**Review by ioannisg — APPROVED** [review]

**Pull request merged**